deepmd/descriptor/se_a.py

```diff
@@ -16,23 +16,58 @@
 from deepmd.utils.graph import load_graph_def, get_tensor_by_name_from_graph
 
 class DescrptSeA ():
-    """DeepPot-SE constructed from all information (both angular and radial) of
-    atomic configurations.
-    
-    The embedding takes the distance between atoms as input.
+    r"""DeepPot-SE constructed from all information (both angular and radial) of
+    atomic configurations. The embedding takes the distance between atoms as input.
+
+    The descriptor :math:`\mathcal{D}^i \in \mathcal{R}^{M_1 \times M_2}` is given by [1]_
+
+    .. math::
+        \mathcal{D}^i = (\mathcal{G}^i)^T \mathcal{R}^i (\mathcal{R}^i)^T \mathcal{G}^i_<
+
+    where :math:`\mathcal{R}^i \in \mathbb{R}^{N \times 4}` is the coordinate
+    matrix, and each row of :math:`\mathcal{R}^i` can be constructed as follows
+
+    .. math::
+        (\mathcal{R}^i)_j = [
+        \begin{array}{c}
+            s(r_{ji}) & x_{ji} & y_{ji} & z_{ji}
+        \end{array}
+        ]
+
+    where :math:`\mathbf{R}_{ji}=\mathbf{R}_j-\mathbf{R}_i = (x_{ji}, y_{ji}, z_{ji})` is 
+    the relative coordinate and :math:`r_{ji}=\lVert \mathbf{R}_{ji} \lVert` is its norm.
+    The switching function :math:`s(r)` is defined as:
+
+    .. math::
+        s(r)=
+        \begin{cases}
+        \frac{1}{r}, & r<r_s \\
+        \frac{1}{r} \{ \frac{1}{2} \cos (\frac{\pi (r - r_s)}{ r_c - r_s} ) +\frac{1}{2} \}, & r_s \leq r<r_c \\
+        0, & r \geq r_c
+        \end{cases}
```

Comment:
We are now using 5-th order polynomial as switcher. It is smooth upto 2nd order derivative. The cosine switcher is only smooth upto the first order derivative. 

https://github.com/njzjz/deepmd-kit/blob/24b433b20e51fc45c7ae587104d00cf732b0e92a/source/lib/include/switcher.h#L77-L92

Reply:
Oh, I don't know that.

Reply:
Updated equation:
![image](https://user-images.githubusercontent.com/9496702/130381227-73f48e9b-3090-4fad-9754-8d0786a0a457.png)